namespaces/systemd.yml

```diff
@@ -151,6 +151,71 @@ namespace:
       description: >
         TODO
 
+    - name: AUDIT_FIELD_ACCT
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_ADDR
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_COMM
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_EXE
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_GRANTORS
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_HOSTNAME
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_OP
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_RES
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_SAUID
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_TERMINAL
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_FIELD_UNIT
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_ID
+      type: string
+      description: >
+        TODO
+
+    - name: AUDIT_TYPE
+      type: string
+      description: >
+        TODO
+
```

Comment:
Are these related to audit system logging?  If so, are they related to aushape?  @spbnick